charts/rancher-webhook/rancher-webhook/0.1.200-rc1+up0.1.2-rc1/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/hidden: "true"
+  catalog.cattle.io/namespace: cattle-system
+  catalog.cattle.io/os: linux
+  catalog.cattle.io/release-name: rancher-webhook
```

Comment:
If we have split, can you add a max of rancher 2.5.99 here. Add a min of 2.5.10 too.

Reply:
`catalog.cattle.io/rancher-version: '>= 2.5.10 <2.5.99'` is added